Automattic/wp-calypso
# Stats: Introducing redirect flow to all Stats pages

Related to #86976 #86929

## Proposed Changes

* Introduce a new component capturing access requirements added to the main Traffic page
* Wrap all pages in the `StatsRedirectFlow` component to avoid page flicking before redirecting

## Testing Instructions



* create a JN and connect it
* navigate to the live branch and apply `stats/checkout-flows-v2` feature flag
* navigate to the stats page and verify that you are redirected to the purchase page
* repeat the process by going to different stats pages like `Insights`, `Subscribers` or summary pages
* click the `I will do it later` button from the purchase page (note, there will be a delay that will use a loader in the future)
* verify that you are redirected to the Traffic page
* verify that you are not redirected to the purchase page for your old blogs

## Pre-merge Checklist
